src/data/map.js

```diff
@@ -1105,7 +1106,7 @@ const getNests = async (minLat, maxLat, minLon, maxLon, nestFilterExclude = null
         args.push(excludedPokemon[i]);
     }
 
-    const results = await dbManual.query(sql, args);
+    const results = await (dbOptions.includes('nest') ? db : dbManual).query(sql, args);
```

Comment:
We want to use manualdb for nests no? Especially since the `dbOptions` that's import is only for scanner tables?
```suggestion
    const results = await (dbOptions.includes('nest') ? dbManual : db).query(sql, args);
```

Reply:
Correct. Since the ternary will return false (assuming they set nests and portals to manualDb in config), then this will result in it reading from the manualDb.